src/containers/Dashboard.vue

```diff
@@ -159,6 +159,24 @@ export default defineComponent({
       showTourEndWindow: false,
       showToolsPage: false,
       isProfileOpen: false,
+      fullCollegeNameMap: new Map<string, string>([
+        ['AS', 'A&S'],
+        ['AS1', 'A&S'],
+        ['AS2', 'A&S'],
+        ['cas', 'A&S'],
+        ['AR', 'AAP'],
+        ['aap', 'AAP'],
+        ['cals', 'CALS'],
+        ['AG', 'CALS'],
+        ['EN', 'CoE'],
+        ['eng', 'CoE'],
+        ['HE', 'HumEc'],
+        ['humec', 'HumEc'],
+        ['IL', 'ILR'],
+        ['ilr', 'ILR'],
+        ['johnson', 'Dyson'],
+        ['BU', 'Dyson'],
+      ]),
```

Comment:
Why is this part of a component's data? It is not part of the Dashboard's state.